utils/dataset_download.py

```diff
@@ -0,0 +1,57 @@
+import os
+from kaggle.api.kaggle_api_extended import KaggleApi
+from dotenv import load_dotenv
+
+def download_and_cleanup_kaggle_dataset():
+    """
+    Downloads a dataset from Kaggle and extracts its contents into the 'data' folder in the root directory.
+    Files specified in the 'DELETE_FILES' environment variable will be removed after extraction.
+
+    This function:
+    1. Loads dataset name and list of files to delete from environment variables using python-dotenv.
+    2. Downloads and unzips the dataset specified in the .env file to the 'data' folder.
+    3. Removes files listed in 'DELETE_FILES' from the 'data' folder after extraction.
+
+    Requirements:
+    - A valid Kaggle API key stored in the file 'kaggle.json' in the directory: `~/.kaggle/` (for Unix-like systems) or `%USERPROFILE%\.kaggle\` (for Windows).
+    - The dataset name should be specified in the .env file under the key 'DATASET'.
+    - Files to be deleted should be specified in the .env file under the key 'DELETE_FILES', separated by commas.
+    """
+    # Ensure fresh environment variables are loaded
+    if 'DELETE_FILES' in os.environ:
```

Comment:
If that is the var from .env, add .env.example so anyone will understand the purpose of this vars.